assets/js/components/TourTooltips.test.js

```diff
@@ -246,6 +246,17 @@ describe( 'TourTooltips', () => {
 		expect( queryByRole( 'alertdialog' ) ).not.toBeInTheDocument();
 	} );
 
+	it( 'should call the callback prop when the tour is started', () => {
```

Comment:
Let's name the test case to be more relevant to what we're verifying.

```suggestion
	it( 'should load Joyride in controlled mode', () => {
```